salesagility/SuiteCRM
# Adding in fix to force a redirect during stage 4 of the upgrade.

Adding in additional section of the commit stage of the upgrade, where we force a redirect to allow for changes to smarty to be picked up. 

## Description
This fixes an issue where upgrades were failing due to a move of smarty to composer. Smarty has already been defined prior to the commit and therefore fails as it cannot find the smarty files once moved. To fix this, a redirect is run to force the new class definition to be picked up. 

## Motivation and Context
Fixes the white screen issue during commit of an upgrade.

## How To Test This
In order to test this, the changes will need to be included into an upgrade pack and go through the standard upgrade process. Ensure that it completes the commit process without white screen. Note that https://github.com/salesagility/SuiteCRM/pull/8257 should also be included to test the full process completes without issue. (This has been tested locally on upgrading 7.11.5->7.11.10).

## Types of changes


### Final checklist
